Lib/turtledemo/forest.py

```diff
@@ -1,14 +1,14 @@
-"""     turtlegraphics-example-suite:
+"""     turtle-example-suite:
 
-             tdemo_forest.py
+        turtledemo/forest.py
 
 Displays a 'forest' of 3 breadth-first-trees
 similar to the one in tree.
 For further remarks see tree.py
 
 This example is a 'breadth-first'-rewrite of
 a Logo program written by Erich Neuwirth. See
-http://homepage.univie.ac.at/erich.neuwirth/
+https://homepage.univie.ac.at/erich.neuwirth/
```

Comment:
Remove this link (German).  It has nothing to do with with the Logo program.  Anyone wanting to know about him should do a normal search.

Reply:
I meant to only remove the quoted weblink, not the credit line "The example ... written by Erich Neuwirth."  Please restore it.

Reply:
Done.